src/clojure/nrepl/middleware/pr_values.clj

```diff
@@ -1,43 +1,67 @@
 (ns nrepl.middleware.pr-values
   {:author "Chas Emerick"}
   (:require
-   [nrepl.middleware :refer [set-descriptor!]])
+    [clojure.string :as str]
```

Comment:
I guess the indentation change was unintentional. I'll assume you're not an Emacs user (or you're using non-default indentation settings). :-)